components/brave_wallet/browser/internal/hd_key_unittest.cc

```diff
@@ -352,8 +419,12 @@ TEST(HDKeyUnitTest, DeriveChildFromPath) {
     std::unique_ptr<HDKey> key = HDKey::GenerateFromExtendedKey(
         "xpub661MyMwAqRbcFtXgS5sYJABqqG9YLmC4Q1Rdap9gSE8NqtwybGhePY2gZ29ESFjqJo"
         "Cu1Rupje8YtGqsefD265TMg7usUDFdp6W1EGMcet8");
-    std::unique_ptr<HDKeyBase> derived_key =
-        key->DeriveChildFromPath("m/3353535/2223/0/99424/4/33");
+    std::unique_ptr<HDKeyBase> derived_key = key->DeriveNormalChild(3353535)
+                                                 ->DeriveNormalChild(2223)
+                                                 ->DeriveNormalChild(0)
+                                                 ->DeriveNormalChild(99424)
+                                                 ->DeriveNormalChild(4)
+                                                 ->DeriveNormalChild(33);
```

Comment:
As far as I understand derivation by path makes sense only for master key generated from seed, not from serialized key. However it still ok to derive children explicitly